packages/react-native-bpk-component-panel/src/__snapshots__/withDivider-test.ios.js.snap

```diff
@@ -329,6 +350,9 @@ exports[`iOS withDivider should render correctly with custom stub style 1`] = `
         Object {
           "flex": 1,
         },
+        Object {
+          "padding": 16,
+        },
```

Comment:
Feels suspect this extra padding has appeared.. does the component look ok?

Reply:
This might be due to a change with how the component processes it e.g. may have removed an array and changed it to pass through differently - component looked ok but will check it

Reply:
So looks like the `bpk-theming` was removing this padding and now we have removed it from `BpkPanel` its brought it back 🤔 Running it now to check what it looks like - might now conflict with a change we made previously to fix RN panel padding